### PR TITLE
chore(Routes): Add the state label to the routes list

### PR DESCRIPTION
**Pull request by mmelko**
Screenshot 2024-04-16 at 16 22 44

### CODE REVIEW

File: packages/hawtio/src/plugins/camel/routes/CamelRoutes.tsx [hunk 1]
```diff
@@ -215,7 +220,9 @@ export const CamelRoutes: React.FunctionComponent = () => {
                   }}
                 />
                 <Td dataLabel='Name'>{route.routeId}</Td>
-                <Td dataLabel='State'>{route.state}</Td>
+                <Td dataLabel='State'>
+                  <RouteStateLabel state={route.state!} />
```
Comment (tadayosi) [inline packages/hawtio/src/plugins/camel/routes/CamelRoutes.tsx:224]: Why `state!`?  I believe it's a non-null assertion, but you don't seem to really assert the state is not null nor undefined. Instead, isn't it better to provide a default value with `route.state ?? <something>`?

**Review by tadayosi — APPROVED** [review]
lgmt, but one comment.

File: packages/hawtio/src/plugins/camel/routes/CamelRoutes.tsx (hunk 1 above)
Comment (mmelko) [reply packages/hawtio/src/plugins/camel/routes/CamelRoutes.tsx:224]: you,re right. Or default unknown state in the Label component.

Comment (tadayosi) [reply packages/hawtio/src/plugins/camel/routes/CamelRoutes.tsx:224]: Or simply leave it empty?

Comment (tadayosi) [reply packages/hawtio/src/plugins/camel/routes/CamelRoutes.tsx:224]: e.g. `{route.state ? <RouteStateLabel state={route.state} /> : ''}`

Comment (mmelko) [reply packages/hawtio/src/plugins/camel/routes/CamelRoutes.tsx]: Well I'd like to have a gray unknown label instead  😇

**Comment by mmelko** [comment]
I also refactored the test in the BlockedExchanges. Looks like there were a change of the behaviour after the upgrade of the react/testing-library in findBy...

**Pull request merged**